### PR TITLE
[3.6] bpo-31948: Fix broken links in msilib docs (GH-4397).

### PR DESCRIPTION
**Pull request by Jesse-Bakker**
(cherry picked from commit 3bda02222aa3783bf85fc3ff8bc042aefd9c4fd3)

https://bugs.python.org/issue31948

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Doc/library/msilib.rst [hunk 1]
```diff
@@ -155,9 +155,10 @@ Database Objects
 
 .. seealso::
 
-   `MSIDatabaseOpenView <https://msdn.microsoft.com/library?url=/library/en-us/msi/setup/msidatabaseopenview.asp>`_
-   `MSIDatabaseCommit <https://msdn.microsoft.com/library?url=/library/en-us/msi/setup/msidatabasecommit.asp>`_
-   `MSIGetSummaryInformation <https://msdn.microsoft.com/library?url=/library/en-us/msi/setup/msigetsummaryinformation.asp>`_
+   `MSIDatabaseOpenView <https://msdn.microsoft.com/en-us/library/windows/desktop/aa370082.aspx>`_
+   `MSIDatabaseCommit <https://msdn.microsoft.com/en-us/library/windows/desktop/aa370075.aspx>`_
+   `MSIGetSummaryInformation <https://msdn.microsoft.com/en-us/library/windows/desktop/aa370301.aspx>`_
+   `MsiCloseHandle <https://msdn.microsoft.com/en-us/library/windows/desktop/aa370067.aspx>`_
```
Comment (Mariatta) [inline Doc/library/msilib.rst]: `Database.Close()` is new in 3.7, so the entry for MsiCloseHandle shouldn't be in 3.6.

**Review by Mariatta — CHANGES_REQUESTED** [review]

**Comment by Jesse-Bakker** [comment]
I have made the requested changes; please review again

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Awesome! Thanks @Jesse-Bakker!

**Pull request merged**